src/EnergyPlus/SQLiteProcedures.hh

```diff
@@ -56,10 +56,10 @@ public:
 	// This allows for testing of private methods in SQLite
 	friend class SQLiteFixture;
 
-	void addScheduleData( int const number, std::string const name, std::string const type, double const minValue, double const maxValue );
+	void addScheduleData( int const number, std::string const & name, std::string const & type, double const minValue, double const maxValue );
```

Comment:
@DeadParrot We can talk in more detail offline and I can walk you through the approach.